pom.xml

```diff
@@ -69,77 +67,53 @@
 			<url>https://repo.jenkins-ci.org/public/</url>
 		</pluginRepository>
 	</pluginRepositories>
-
+	<dependencyManagement>
+		<dependencies>
+			<dependency>
+				<groupId>io.jenkins.tools.bom</groupId>
+				<artifactId>bom-2.346.x</artifactId>
+				<version>1607.va_c1576527071</version>
+				<scope>import</scope>
+				<type>pom</type>
+			</dependency>
+		</dependencies>
+	</dependencyManagement>
 	<dependencies>
 		<dependency>
 			<groupId>org.jenkins-ci.plugins</groupId>
 			<artifactId>jackson2-api</artifactId>
-			<version>2.8.11.3</version>
 		</dependency>
 
 		<!-- dependencies on Jenkins Pipeline plugins -->
 		<dependency>
 			<groupId>org.jenkins-ci.plugins.workflow</groupId>
 			<artifactId>workflow-job</artifactId>
-			<version>1.14</version>
 		</dependency>
 		<dependency>
 			<groupId>org.jenkins-ci.plugins.workflow</groupId>
 			<artifactId>workflow-cps</artifactId>
-			<version>1.14</version>
 		</dependency>
 		<dependency>
 			<groupId>org.jenkins-ci.plugins.workflow</groupId>
 			<artifactId>workflow-step-api</artifactId>
-			<version>1.14</version>
 		</dependency>
 		<!-- Required due to JENKINS-51013 in plugin compat tester -->
-		<dependency>
+		<!-- <dependency>
```

Comment:
Does this mean that this dependency is no longer required? If so, please remove the now commented blocks.

Reply:
Seems the mentioned issue was fixed and doesn't appear using more recent parent. There is also no reason to have a hard dependency on this plugin

Reply:
thx